csi/src/node/nvmfutil.rs

```diff
@@ -60,13 +59,14 @@ fn uuid_from_str(s: &str) -> String {
     // } hack
 }
 
-fn wait_for_path_to_exist(uuid: String, max_retries: i32) -> Option<String> {
-    let second = time::Duration::from_secs(1);
-    let mut retries: i32 = 0;
+fn wait_for_path_to_exist(
+    uuid: String,
+    timeout: time::Duration,
+    max_retries: u32,
+) -> Option<String> {
+    let mut retries: u32 = 0;
     let now = time::Instant::now();
     while retries < max_retries {
```

Comment:
if max_**re**tries is 0 we don't even **try**?

Reply:
fixed